transcom/mymove
# B 18059 int - download link for AOA packet office user

## [B-18059](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A870620)

## Summary

As a service counselor when you approve an Advance request a link is displayed on the PPM shipment card for downloading the AOA packet. The link doesn't work with this story but will be created and downloaded as part of a future story. This is to verify the 

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Create a PPM shipment as a customer and ask for an advance
2. Login as the service counselor and approve the advance
3. There should be a link to download the advance on the PPM shipment card

### Frontend


## Screenshots
